bigchaindb/web/views/statuses.py

```diff
@@ -0,0 +1,62 @@
+"""This module provides the blueprint for the statuses API endpoints.
+
+For more information please refer to the documentation on ReadTheDocs:
+ - https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/
+   http-client-server-api.html
+"""
+from flask import current_app
+from flask_restful import Resource, reqparse
+
+from bigchaindb.web.views.base import make_error
+
+
+class StatusApi(Resource):
+    def get(self):
+        """API endpoint to get details about the status of a transaction or a block.
+
+        Return:
+            A ``dict`` in the format ``{'status': <status>}``, where
+            ``<status>`` is one of "valid", "invalid", "undecided", "backlog".
+        """
+        parser = reqparse.RequestParser()
+        parser.add_argument('tx_id', type=str)
+        parser.add_argument('block_id', type=str)
+
+        args = parser.parse_args(strict=True)
+
```

Comment:
I'd prefer if we extract `tx_id` and `block_id` as Python variables here already.

We can then:

- match the 400 error chase in a more readable fashion: `if tx_id is None and block_id is None: return error`
- We don't have to access `args` throughout the whole function call.

Your call if you want to change that, but I think it would increase readability.